mafContrib/TDEsPopMetric.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+from lsst.sims.maf.utils import m52snr
+import lsst.sims.maf.metrics as metrics
+import os
+from lsst.sims.utils import uniformSphere
+import lsst.sims.maf.slicers as slicers
+import glob
+from lsst.sims.photUtils import Sed
+
+
+__all__ = ['tde_lc', 'TDEsPopMetric', 'TDEsPopSlicer']
+
```

Comment:
Python class names are generally UpperCase - https://www.python.org/dev/peps/pep-0008/#class-names
I think if it's lower case it implies that the thing is a function. 
If the point of tde_lc is to really be a class and hold the lightcurve data (instead of returning it to the user to use and interpolate), then it should probably be capitalized so that it's immediately clear it's a class not a function.